components/match2/wikis/osu/match_group_input_custom.lua

```diff
@@ -0,0 +1,459 @@
+---
+-- @Liquipedia
+-- wiki=osu
+-- page=Module:MatchGroup/Input/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Opponent = require('Module:Opponent')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local TypeUtil = require('Module:TypeUtil')
+local Variables = require('Module:Variables')
+local DateExt = require('Module:Date/Ext')
+local Streams = require('Module:Links/Stream')
+local Match = require('Module:Match')
+
+local MatchGroupInput = Lua.import('Module:MatchGroup/Input', {requireDevIfEnabled = true})
+
+local ALLOWED_VETOES = {'decider', 'pick', 'ban', 'defaultban', 'protect'}
+local ALLOWED_STATUSES = { 'W', 'FF', 'DQ', 'L', 'D' }
+local MAX_NUM_OPPONENTS = 8
```

Comment:
sure about that?
MatchSummary display for that is not supported ...

Reply:
If this is for `local MAX_NUM_OPPONENTS = 8`, likely copyover. Its mostly 4 per match/team at a time

Reply:
that variables states how many teams (opponents) a match has at maximum
it is not stating how many players are in a team